test/functional/apps/discover/_discover.js

```diff
@@ -160,7 +160,7 @@ export default function ({ getService, getPageObjects }) {
 
         const newDurationHours = await PageObjects.timePicker.getTimeDurationInHours();
         expect(Math.round(newDurationHours)).to.be(3);
-        const rowData = await PageObjects.discover.getDocTableIndex(1);
+        const rowData = await PageObjects.discover.getDocTableField(1);
```

Comment:
In case of failure, we were getting a long string with all fields. To simplify analysis `getDocTableField` returns the date only.